src/jwplayer/utils/useAdEngineComplete.ts

```diff
@@ -5,14 +5,16 @@ import { race, timer } from 'rxjs';
 import { first } from 'rxjs/operators';
 import { recordVideoEvent, VIDEO_RECORD_EVENTS } from 'jwplayer/utils/videoTimingEvents';
 
-export default function useAdComplete(): boolean {
+export default function useAdEngineComplete(): boolean {
 	const [adComplete, setAdComplete] = useState(false);
 
 	useEffect(() => {
 		recordVideoEvent(VIDEO_RECORD_EVENTS.JW_PLAYER_AD_ENG_OPT_IN_LISTEN_START);
+		// TODO: we probably can remove this step as AdEngine won't send "[Ad Engine] Setup JWPlayer" before the consent
```

Comment:
This comment seems more like a personal reminder, and it's not clear what action needs to be taken. If we have the opportunity to address this issue, why not go ahead and do so? If there is a related Jira ticket or GitHub issue, it would be helpful to include a link here.

Reply:
Yeah, it's something like that. A reminder ;) We can't take actions now and we will after we conduct the experiment.